README.md

```diff
@@ -11,24 +11,49 @@ This project is for OSIS which integrates [Scality RING](https://www.scality.com
 
 ## Running the Application
 
-### Command to run Docker Image
-A user needs to be logged into docker using URL: registry.scality.com.
-```sh
-    $ docker login -u <username in registry.scality.com> registry.scality.com
-```
-Once logged images can be pulled and run locally from either dev namespace using the hash or production namespace.
+### Commands to run Docker Image
+1. Docker images can be pulled locally from either dev namespace using the hash or production namespace.
+
+    Dev:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
+    ```
+    Production image:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality/vmware-ose-scality:<tag>
+    ```
+1. Generate a self-signed SSL certificate and store it in a binary PKCS#12 format file with extension `.p12` file. (Refer [here](#To-generate-PKCS12-file-for-self-signed-SSL-certificate))
+
+1. Create an environment variables file `application.properties`
+    - Sample `application.properties` file can be found [here](src/main/resources/application.properties).
+    - Update `server.ssl.key-store-password` and `server.ssl.key-alias` with key-store password and key-store alias of the `.p12` file respectively.
+    
+1. To run the docker image locally from either dev namespace using the hash or production namespace with the environment file `application.properties`
+
+    Dev:
+    ```sh
+        $ docker run --env-file application.properties -it -p8443:8443 -v <absolute_path_to_.p12_file>:/app/lib/osis.p12:z registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
+    ```
+    Production image:
+    ```sh
+        $ docker run --env-file application.properties -it -p8443:8443 <absolute_path_to_.p12_file>:/app/lib/osis.p12:z registry.scality.com/vmware-ose-scality/vmware-ose-scality:<tag>
+    ```
+
 
-Dev:
-```sh
-    $ docker pull registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
-    $ docker run --env-file application.properties -it -p8443:8443 registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
+#### To generate PKCS12 file for self-signed SSL certificate
+**Generate a `.p12` format file with self-signed SSL certificate using Keytool.**
+```shell
+keytool -genkeypair -keyalg RSA -alias <alias_name> -keystore <key-store_file>.p12 \
+-storepass <key-store_password> -validity <days> -keysize <size_in_bytes> \
+-dname "CN=<common_name>, OU=<organization_unit>, O=<organization>, L=<city>, ST=<state>, C=<country>" \
+-storetype pkcs12
```

Comment:
I would split after each line and add spaces, it works fine to copy paste and breaks up the input more when viewing it.
Same below if you agree.

```suggestion
keytool \
  -genkeypair \
  -keyalg RSA \
  -alias <alias_name> \
  -keystore <key-store_file>.p12 \
  -storepass <key-store_password> \
  -validity <days> \
  -keysize <size_in_bytes> \
  -dname "CN=<common_name>, OU=<organization_unit>, O=<organization>, L=<city>, ST=<state>, C=<country>" \
  -storetype pkcs12
```

Reply:
Covered this in e3ae684ae588760d0b37f12fe614eda2c282a644